### PR TITLE
leaky_relu added to jax frontend

### PR DESCRIPTION
**Pull request by VikramxD**

### CODE REVIEW

File: ivy/functional/frontends/jax/nn/non_linear_activations.py [hunk 1]
```diff
@@ -6,3 +6,17 @@ def relu(x):
...
+
+
+def leaky_relu(x, alpha=0.01):
+    return ivy.leaky_relu(x, alpha=alpha)
+
+
+leaky_relu.unsupported_dtypes = {"torch": ("float16", "bfloat16")}
+
+
+def gelu(x):
```
Comment (sherry30) [inline ivy/functional/frontends/jax/nn/non_linear_activations.py:18]: remove this

**Review by sherry30 — CHANGES_REQUESTED** [review]
Hi,

I think you've accidentally added `gelu`, can you kindly remove that.
other than that if you see the documentation of `leaky_relu` [here](https://jax.readthedocs.io/en/latest/_autosummary/jax.nn.leaky_relu.html#jax.nn.leaky_relu), the argument is called `negative_slope` and not `alpha`. so you will have to change that argument name.

Thanks

**Comment by VikramxD** [comment]
pr for #2823 from the todolist issue #1515  

1. Implemented leaky_relu in  ivy/functional/frontends/jax/nn/non_linear_activations.py
2. added tests appropriate to leaky_relu functions in ivy_tests/test_ivy/test_frontends/test_jax/test_activations.py

**Comment by VikramxD** [comment]
@sherry30 this pr is ready for re review please do tell if anything else needs to be changed

File: ivy/functional/frontends/jax/nn/non_linear_activations.py (hunk 1 above)
Comment (VikramxD) [reply ivy/functional/frontends/jax/nn/non_linear_activations.py]: removed

**Review by sherry30 — CHANGES_REQUESTED** [review]
Hi,

Can you also add `negative_slope` argument in your test.
Ideally the test function should test all arguments of the function.

Thanks.

**Comment by VikramxD** [comment]
@sherry30 changes done accordingly

**Pull request merged**